hi.js

```diff
@@ -97,17 +97,17 @@ settings.experimental.header आजमाइशी
 settings.experimental.aimFreezeFix लक्ष्य के दौरान फ्रीज को ठीक करें
 settings.experimental.mouseFlickFix लक्ष्य के दौरान झटके को ठीक करें
 settings.experimental.shaderRendering शेडर रैडंर
-settings.client.header Client
-settings.client.unlimitedFrames Unlimited FPS
-settings.client.gameCap Game Capture
-settings.client.experimental Experimental Features
-settings.client.discordRPC Discord RPC
-settings.client.resourceSwapper Resource Swapper
-settings.client.clearCache Clear Cache
-settings.client.resetSettings Reset Client Settings
-settings.client.acceleratedCanvas Accelerated Canvas
-settings.client.angleBackend ANGLE Graphics Backend
-settings.client.colorProfile Color Profile
+settings.client.header Client क्लाइंट
+settings.client.unlimitedFrames Unlimited FPS असीमित FPS
+settings.client.gameCap Game Capture खेल कैप्चर
+settings.client.experimental Experimental Features आजमाइशी फ़ीचर
+settings.client.discordRPC Discord RPC डिस्कॉर्ड आर पी सी
+settings.client.resourceSwapper Resource Swapper साधन अदला-बदली
+settings.client.clearCache Clear Cache Cache साफ़
+settings.client.resetSettings Reset Client Settings रीसेट क्लाइंट सेटिंग
```

Comment:
@akaRupture can you please remove the English components where you don't need it?

Reply:
across all your changes etc.

Reply:
done